docs/time_series_performance_benchmarks.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
<p>Are the bbox dimensions the size in pixels or degrees? I'm assuming pixels but then curious why we have 360x180 when the other options are square and also a factor of standard tile sizes, in pixels.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/titiler-cmr/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>